OneCommunityGlobal/HighestGoodNetworkApp
# Shengjie unit test codes for TagSent in WBS

# Description
Created unit tests for the TagSent component in WBS. Checked for functionalities and edge cases.

## Related PRS (if any):
None

## Main changes explained:
- Created tests for check props
- Tested components rendering
- Checked for input types

## How to test:
1. check into current branch
2. do `npm install` and `npm test TagSent.WBS.test.js` to run this PR locally

## Screenshots or videos of changes:
<img width="729" alt="screenshot" src="https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/56333697/1e4932b4-f8e1-4e9a-915c-bcd7ee6b2c83">


## Note:
Tried to test the case of missing inputs, but the original function may lack the checks on inputs validities. Kept the codes in the comments still in case of future needs.
